dipy/tracking/utils.py

```diff
@@ -420,6 +431,77 @@ def seeds_from_mask(mask, density=[1, 1, 1], voxel_size=None, affine=None):
 
     return seeds
 
+def random_seeds_from_mask(mask, seeds_per_voxel=1, affine=None):
+    """Creates randomly placed seeds for fiber tracking from a binary mask.
+
+    Seeds points are placed randomly distributed in all voxels of ``mask`` which
+    are ``True``. This function is essentially similar to ``seeds_from_mask()``,
+    with the difference that instead of evenly distributing the seeds, it randomly
+    places the seeds within the voxels specified by the ``mask``
+
+    Parameters
+    ----------
+    mask : binary 3d array_like
+        A binary array specifying where to place the seeds for fiber tracking.
+    seeds_per_voxel : int
+        Specifies the number of seeds to place in each voxel.
+    affine : array, (4, 4)
+        The mapping between voxel indices and the point space for seeds. A
+        seed point at the center the voxel ``[i, j, k]`` will be represented as
+        ``[x, y, z]`` where ``[x, y, z, 1] == np.dot(affine, [i, j, k , 1])``.
+
+    See Also
+    --------
+    seeds_from_mask
+
+    Raises
+    ------
+    ValueError
+        When ``mask`` is not a three-dimensional array
+
+    Examples
+    --------
+    >>> mask = np.zeros((3,3,3), 'bool')
+    >>> mask[0,0,0] = 1
+    >>> random_seeds_from_mask(mask, seeds_per_voxel=1, voxel_size=[1,1,1])
```

Comment:
This doctest is currently failing, because the function does not have a `voxel_size` kwarg. Is knowing the voxel size important for this function? If it's not, can you remove this kwarg from the doctest?